docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt

```diff
@@ -1,16 +1,8 @@
-cmake_minimum_required(VERSION 3.8)
-
-project(test_package C) # if the project is pure C
-project(test_package CXX) # if the project uses c++
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-find_package(package REQUIRED CONFIG)
-
-# Re-use the same source file from test_package folder
-add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
-# don't link to ${CONAN_LIBS} or CONAN_PKG::package
-target_link_libraries(${PROJECT_NAME} PRIVATE package::package)
-# in case the target project requires a C++ standard
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
+                 ${CMAKE_CURRENT_BINARY_DIR}/binaries/)
```

Comment:
I think it makes more sense to name the `binaries` directory `test_v1_package` so that we know what that belongs to in the build directory.

Reply:
Done